safe_web_api_playground/static/js/code-snippets/safeApp.js

```diff
@@ -0,0 +1,152 @@
+module.exports = {
+  safeApp: {
+
+    initialise: () => {
+
+      // Welcome! Get started by running this snippet.
+      // Then open the `safeApp` module on the left-hand side of the screen.
+      // Reference documentation: http://docs.maidsafe.net/beaker-plugin-safe-app/
+      // When not focused in this text box, simply use your `Enter` key at any time to run code snippets.
+
+      let appInfo = {
+        id: 'net.maidsafe.api_playground.webclient.10',
+        name: 'SAFE browser API playground',
```

Comment:
Maybe change this to `SAFE DOM API Playground`? otherwise it sounds like it's a browser's API rather than SAFE API